lib/json2csv.js

```diff
@@ -99,14 +103,14 @@ var createColumnContent = function(params, str, callback) {
 
       params.fields.forEach(function(field_element) {
         if (data_element.hasOwnProperty(field_element)) {
-          line += JSON.stringify(data_element[field_element]);
+          line += JSON.stringify(data_element[field_element]).replace(/\"/g, params.quotes);
```

Comment:
Yup, you're right.

Maybe we could check if first & last char are `"` and then replace these.

What do you think?